{"uuid":"34bed733-e50e-4123-8366-65794d37ce7e","name":"Default test","children":["fce39568-da23-4748-a69f-584f6a0e9a68","4d391dd5-851e-4a8f-98df-3932933ca97b","a8ca245f-91db-4fed-83f1-4cd5328e4d81","348d66b4-6f29-4a14-b38b-24cbf5991b1f","efb24ac4-717b-4c88-ba7c-495536bc54e6","2ea7ae27-93c1-4977-9cc6-11916c2efd55","f600480b-8981-4733-907b-88946c20a721","0e976743-fdb8-4cba-ad6a-5e5a9064da93"],"befores":[{"name":"setUp","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"chromedriver located at path/to/chromedriver.exe, but invalid\nFor documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/\nBuild info: version: '4.12.1', revision: '8e34639b11'\nSystem info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.8.1'\nDriver info: driver.version: ChromeDriver","trace":"org.openqa.selenium.remote.NoSuchDriverException: chromedriver located at path/to/chromedriver.exe, but invalid\nFor documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/\nBuild info: version: '4.12.1', revision: '8e34639b11'\nSystem info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.8.1'\nDriver info: driver.version: ChromeDriver\r\n\tat org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:45)\r\n\tat org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:13)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.generateExecutor(ChromeDriver.java:99)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:88)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:83)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:51)\r\n\tat runner.FeatureRunner.setUp(FeatureRunner.java:21)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)\r\n\tat org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:663)\r\n\tat org.testng.TestRunner.beforeRun(TestRunner.java:652)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1200)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1114)\r\n\tat org.testng.TestNG.run(TestNG.java:1082)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1699666844508,"stop":1699666844714}],"afters":[{"name":"tearDown","status":"broken","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1699666846035,"stop":1699666846036}],"start":1699666844458,"stop":1699666846039}